web/src/components/NumberDisplay.tsx

```diff
@@ -9,6 +9,14 @@ interface INumberDisplay {
   isCurrency?: boolean; //currency units are shown in front
 }
 
+const getFormattedValue = (value: number, decimals: number) => {
+  const withFixedDecimals = value % 1 !== 0 ? value.toFixed(decimals) : value.toFixed(0);
+  if (withFixedDecimals === `0.${"0".repeat(decimals)}` && value > 0) {
+    return `< 0.${"0".repeat(decimals - 1)}1`;
+  }
```

Comment:
`withFixedDecimals` can be a negative number, we can just take the absolute of `value`
```suggestion
  if (Math.abs(value).toFixed(decimals) === `0.${"0".repeat(decimals)}` && Math.abs(value) > 0) {
    return `< 0.${"0".repeat(decimals - 1)}1`;
  }
```